### PR TITLE
Add Astral Credits (XAC) to Token List

### PR DESCRIPTION
**Pull request by HelloMokuzai**
Howdy!

Requesting to add Astral Credits (XAC) to the Songbird Network (Chain ID: 19) token list.

- Logo's have been added to [/tokens](https://github.com/pangolindex/tokens/pull/288)
- Token contract verified on [Songbird Explorer](https://songbird-explorer.flare.network/address/0x61b64c643fCCd6ff34Fc58C8ddff4579A89E2723)
- Website: https://astralcredits.xyz/

Thank you!
MKZI⚡

### CODE REVIEW

File: pangolin.tokenlist.json [hunk 1]
```diff
@@ -281,6 +281,14 @@
...
       "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/19/0xb1D60fA6FABd9A2Ad89960d070350552E47fb0D6/logo_24.png"
     },
+    {
+      "chainId": 19,
+      "address": "0x61b64c643fCCd6ff34Fc58C8ddff4579A89E2723",
+      "decimals": 18,
+      "name": "Astral Credits",
+      "symbol": "XAC",
+      "logoURI": "https://raw.githubusercontent.com/pangolindex/tokens/main/assets/assets/19/0x61b64c643fCCd6ff34Fc58C8ddff4579A89E2723/logo_24.png"
+    }
```
Comment (SarjuHansaliya) [inline pangolin.tokenlist.json:291]: there is issue with json here, looks like need comma

**Review by HelloMokuzai — COMMENTED** [review]
Resolved, thank you.

File: pangolin.tokenlist.json (hunk 1 above)
Comment (HelloMokuzai) [reply pangolin.tokenlist.json]: Updated in new commit.

**Comment by HelloMokuzai** [comment]
Merged via #355. Closing this PR.

**Pull request closed without merging**